cppcache/include/geode/PoolFactory.hpp

```diff
@@ -533,9 +533,10 @@ class _GEODE_EXPORT PoolFactory {
 
   ~PoolFactory();
 
+  PoolFactory(const PoolFactory&);
```

Comment:
If the copy is trivial, as it should be just default, inline and include I header.

Reply:
Yup. Done.